docs/hugo/content/tools/asoctl.md

```diff
@@ -94,6 +102,34 @@ Append that directory to your `PATH` if desired.
 {{% /tab %}}
 {{< /tabpane >}}
 
+## Template
+
+The `template` commands helps render a YAML file from the raw ASO published with each release.
+
+```bash
+template creates a YAML file from the specified ASO yaml template
+
+Usage:
+  asoctl template <path to ASO YAML> [flags]
```

Comment:
Lots of thoughts.

Suggest the command should be `asoctl export template`, both to allow for exporting of other things, and to follow the approach of `asoctl <verb> <noun>` that we standardised on originally.

Given the number of potential URLs from which to download the required YAML is very low, would be nice to have less verbose options. As discussed offline, maybe

`--source <file|url>` to support the current behaviour
`--tag <tag>` to allow users to specify just the bit of interest (e.g. `asoctl export template --tag v2.7.0`)

Reply:
Can `--crd-pattern` be repeated? If so, include an example of that too?

Reply:
> Suggest the command should be asoctl export template, both to allow for exporting of other things, and to follow the approach of asoctl <verb> <noun> that we standardised on originally.

I've done this but I actually think it's somewhat awkward how we've got it today. If you look for example at `asoctl import` it's documented as: "Imports ARM resources to YAML files containing ASO custom resource definitions" - which isn't really a "generic" import, it's the description of the operation inside `import`.

It's not clear to me what I should write for the `export` documentation... The command is supposed to be in theory generic (could export many things) but it seems difficult to write a useful description of it that doesn't have intimate knowledge about what the subcommands are.

Reply:
Added repeated `--crd-pattern` examples.

Reply:
swapped `--tag` to `--version` as I think that's slightly clearer? LMK what you think.

Reply:
I'm happy with that. I guess `tag` is more of an implementation detail anyway.